Dockerfile.tools

```diff
@@ -9,6 +9,8 @@ COPY . .
 
 RUN yum -y update && yum -y install git make python2 python2-pip gcc
 
+RUN yum install -y epel-release && yum -y install jq
```

Comment:
I should have caught this before. Why can't you fold this into the previous line? `yum -y install git make python2 python2-pip gcc epel-release` && `yum -y install jq`.

I am guessing the `jq` install is in a separate line because `jq` is part of `epel-release` repo. If yes, then please add a comment about it.